hpx/runtime/actions/basic_action.hpp

```diff
@@ -62,14 +62,12 @@ namespace hpx { namespace actions
             HPX_MOVABLE_BUT_NOT_COPYABLE(continuation_thread_function);
 
         public:
-            template <typename F_, typename ...Ts_>
             explicit continuation_thread_function(
-                    std::unique_ptr<continuation> cont,
-                    naming::address::address_type lva, F_&& f, Ts_&&... vs)
+                std::unique_ptr<continuation> cont,
+                naming::address::address_type lva, F&& f, Ts&&... vs)
```

Comment:
I am not sure if this change is correct. If I am not mistaken, ```F&&``` and ```Ts&&...``` are rvalue reference and not forwarding references in that context, correct?